CP04OSSM/CP04OSSM_R00001_ingest.csv

```diff
@@ -1,43 +1,43 @@
+#mi.dataset.driver.optaa_dj.dcl.optaa_dj_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00001/cg_data/dcl27/OPTAA_snACS165/*.optaa1.log,CP04OSSM-RID27-01-OPTAAD000,recovered_host,Not Expected,Blank raw data files
+#mi.dataset.driver.flort_dj.dcl.flort_dj_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00001/cg_data/dcl27/FLORT_sn1221/*.flort.log,CP04OSSM-RID27-02-FLORTD000,recovered_host,Not Expected,Blank raw data files
+#mi.dataset.driver.ctdbp_cdef.dcl.ctdbp_cdef_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00001/cg_data/dcl27/CTDBP_sn16-50004/*.ctdbp1.log,CP04OSSM-RID27-03-CTDBPC000,recovered_host,Not Expected,Blank raw data files
+mi.dataset.driver.ctdbp_cdef.ctdbp_cdef_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00001/cg_data/dcl27/CTDBP_sn16-50004/SBE*.hex,CP04OSSM-RID27-03-CTDBPC000,recovered_inst,Available,
+#mi.dataset.driver.dosta_abcdjm.dcl.dosta_abcdjm_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00001/cg_data/dcl27/DOSTA_sn375/*dosta1.log,CP04OSSM-RID27-04-DOSTAD000,recovered_host,Not Expected,Blank raw data files
+#mi.dataset.driver.cg_cpm_eng.cpm.cg_cpm_eng_cpm_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00001/cg_data/irid2shore/cpm_status*.txt,CP04OSSM-SBC11-00-CPMENG000,recovered_host,Not Expected,no recovered ENG files
+#mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00001/cg_data/dcl11/syslog/*.syslog.log,CP04OSSM-SBD11-00-DCLENG000,recovered_host,Not Expected,no recovered ENG files
+#mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00001/cg_data/dcl11/mopak_*/*.mopak.log,CP04OSSM-SBD11-01-MOPAK0000,recovered_host,Not Expected,no recovered instrument data
+#mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00001/cg_data/dcl11/Hydrogen_sn/*.hyd1.log,CP04OSSM-SBD11-02-HYDGN0000,recovered_host,Not Expected,no recovered instrument data
+#mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00001/cg_data/dcl11/METBK-Log_*/*.metbk*.log ,CP04OSSM-SBD11-06-METBKA000,recovered_host,Not Expected,no recovered instrument data
+#mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00001/cg_data/dcl12/syslog/*.syslog.log,CP04OSSM-SBD12-00-DCLENG000,recovered_host,Not Expected,no recovered ENG files
+#mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00001/cg_data/dcl12/Hydrogen_sn/*.hyd2.log,CP04OSSM-SBD12-03-HYDGN0000,recovered_host,Not Expected,no recovered instrument data
+#mi.dataset.driver.pco2a_a.dcl.pco2a_a_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00001/cg_data/dcl12/PCO2A*/PCO2A*.log,CP04OSSM-SBD12-04-PCO2AA000,recovered_host,Not Expected,no recovered instrument data
+#mi.dataset.driver.zplsc_c.zplsc_c_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00001/cg_data/dcl37/ZPLSC_sn*/DATA/*/*.01A,CP04OSSM-MFD37-07-ZPLSCC000,recovered_inst,Not Deployed,
```

Comment:
Filled in any filemasks and parsers that were missing, even in cases where we aren't expecting data (for consistency....)

Reply:
Looks good.  Suggest moving the ZPLSC recovered instrument line up the other ZPLSC line.
We don't want to ingest ADPC pd8 data.